index.html

```diff
@@ -15,7 +15,7 @@
     --> <title>Ya Nació el Nenu?</title>
     <style>
       html {
-        background: #ff0050;
+        background: #0cd36e;
```

Comment:
cant you use DS tokens instead?

Reply:
Sorry, the usage of acronyms is prohibited in this organization.